code/modules/reagents/reagent_containers.dm

```diff
@@ -349,8 +349,13 @@ var/list/LOGGED_SPLASH_REAGENTS = list(FUEL, THERMITE)
 			reagents.reaction(user, INGEST)
 			spawn(5)
 				if(reagents)
-					reagents.trans_to(user, amount_per_imbibe)
-
+					if(user.get_stomach())
+						var/datum/organ/internal/stomach/S = user.get_stomach()
+						reagents.trans_to(S.get_reagents(), amount_per_imbibe) // transfer to stomach
+					else if(istype(user, /mob/living/carbon/human))
+						reagents.trans_to(user, amount_per_imbibe / 4) // greatly reduced reagent absorption without a stomach
```

Comment:
You're dividing by 4 the effectiveness of all medicine, and poisons, to races without a stomach

Really ?

Reply:
The intention was to limit the effectiveness of medicine if you were a member of a race that had a stomach but didn't have a stomach as an individual. Thanks for pointing that out, I'll correct it.

Reply:
All good